types/types.go

```diff
@@ -334,3 +334,37 @@ type AccessLog struct {
 	FilePath string `json:"file,omitempty" description:"Access log file path"`
 	Format   string `json:"format,omitempty" description:"Access log format: json | common"`
 }
+
+// HasCustomHeadersDefined checks to see if any of the custom header elements have been set
+func (h *Headers) HasCustomHeadersDefined() bool {
```

Comment:
you can use `(h Headers)` instead of `(h *Headers)` because don't update `h` fields.